pkg/driver/controller.go

```diff
@@ -29,7 +29,7 @@ import (
 )
 
 func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest) (*csi.CreateVolumeResponse, error) {
-	glog.V(4).Infof("CreateVolume: called with args %#v", req)
+	glog.V(4).Infof("CreateVolume: called with args %+v", *req)
```

Comment:
Wondering what's the reason of doing pointer dereference here? Is it because "%+v" doesn't work with pointer?

Reply:
That's right, if a field in `req` is a pointer, its value wouldn't make it to the log message.